src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -41,75 +35,48 @@
 
 
     private final UserPrefs userPrefs;
-    private AlfredStorage storage = null;
+    private final Alfred alfred;
 
     // EntityLists
-    private FilteredList<Participant> participantList;
-    private FilteredList<Team> teamList;
-    private FilteredList<Mentor> mentorList;
+    private FilteredList<Participant> participantFilteredList;
+    private FilteredList<Team> teamFilteredList;
+    private FilteredList<Mentor> mentorFilteredList;
```

Comment:
Is there a need to remove the original ParticipantList object?
Is it possible to instead have both participantList and participantFilteredList at the top level?
Because once `participantList` etc is removed, everything on `ModelManager` breaks. 

Also, from this [link](https://docs.oracle.com/javase/8/javafx/api/javafx/collections/transformation/FilteredList.html), seems like once the ObservableList inside `XList.java` is modified, the changes are immediately reflected in the filtered list.